Description: Skip examples that make us run out of memory.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-14

--- a/M2/Macaulay2/packages/TateOnProducts.m2
+++ b/M2/Macaulay2/packages/TateOnProducts.m2
@@ -5959,9 +5959,6 @@
       for i from min RM to max RM list (rank RM_i)
   Text
       RM looks complicated since it is consisted of universal bundles on P^6, which are of high rank.
-  Example
-      retTable=time actionOnDirectImage(I,M,matrix vd);
-      keys retTable
   Text
       We see that 0 is the only key, in other words, there is no other R^i vd_{*} except i=0.
       To see whether it gives an action on S/J, we can use the test function isAction.
@@ -5969,8 +5966,6 @@
       Note that list retTable#i is consisted of maps of chain complexes R^i(\pi \cdot phi)_{*}(y_j) : C\to C(1) where 
       C represents the direct image R^i(\pi \cdot phi)_{*}F. In general, it does not give a right action on C itself. 
       The induced maps on cohomology groups provide a right action.
-  Example
-      isAction(J, apply(dim S, i->prune HH^0 retTable#0#i))
   Text
       The following is a little more complicated example with nontrivial higher direct images. 
       Let X be the product of two quartic curves C, and f : X \to C be the second projection. 
